MonoGame.Framework/Game.cs

```diff
@@ -303,10 +303,12 @@ internal bool Initialized
 
         #region Events
 
+        private readonly ExitingEventArgs _exitingEventArgs = new ExitingEventArgs();

```

Comment:
I assume this is some sort of memory optimization?

Seems unnecessary and complicates the PR.  It isn't like we get dozens of exit events each frame.

Reply:
Yes, just to avoid allocating. I agree it's unnecessary in this case.